dev/mhmohona/Startup task - Train and test a classification model.ipynb

```diff
@@ -0,0 +1,1912 @@
+    "\n",
+    "X18-X23: Amount of previous payment (NT dollar). X18 = amount paid in September, 2005; X19 = amount paid in August, 2005; . . .;X23 = amount paid in April, 2005.\n",
+    "\n",
+    "\n",
+    "#### Attribute Descripition\n",
+    "\n",
+    "Now if we look at the .csv file, we can see there are 25 variables:\n",
+    "\n",
+    "**ID**\n",
+    "- *id:* ID of each client\n",
+    "\n",
+    "**Numeric Variables**\n",
+    "- *limit_bal*: Amount of given credit in NT dollars (includes individual and family/supplementary credit\n",
+    "- *bill_amt1*: Amount of bill statement in September, 2005 (NT dollar)\n",
+    "- *bill_amt2*: Amount of bill statement in August, 2005 (NT dollar)\n",
+    "- *bill_amt3*: Amount of bill statement in July, 2005 (NT dollar)\n",
+    "- *bill_amt4*: Amount of bill statement in June, 2005 (NT dollar)\n",
+    "- *bill_amt5*: Amount of bill statement in May, 2005 (NT dollar)\n",
+    "- *bill_amt6*: Amount of bill statement in April, 2005 (NT dollar)\n",
+    "- *pay_amt1*: Amount of previous payment in September, 2005 (NT dollar)\n",
+    "- *pay_am2*: Amount of previous payment in August, 2005 (NT dollar)\n",
+    "- *pay_am3*: Amount of previous payment in July, 2005 (NT dollar)\n",
+    "- *pay_am4*: Amount of previous payment in June, 2005 (NT dollar)\n",
+    "- *pay_am5*: Amount of previous payment in May, 2005 (NT dollar)\n",
+    "- *pay_am6*: Amount of previous payment in April, 2005 (NT dollar)\n",
+    "\n",
+    "**Ordinal Variables**\n",
+    "- *pay_0*: Repayment status in September, 2005 (-1=pay duly, 1=payment delay for one month, 2=payment delay for two months,8=payment delay for eight months, 9=payment delay for nine months and above)\n",
+    "- *pay_2*: Repayment status in August, 2005 (scale same as above)\n",
+    "- *pay_3*: Repayment status in July, 2005 (scale same as above)\n",
+    "- *pay_4*: Repayment status in June, 2005 (scale same as above)\n",
+    "- *pay_5*: Repayment status in May, 2005 (scale same as above)\n",
+    "- *pay_6*: Repayment status in April, 2005 (scale same as above)\n",
+    "\n",
+    "**Categorical Variables**\n",
+    "- *sex*: Gender (1 = male, 2 = female)\n",
+    "- *education*: (1 = graduate school, 2 = university, 3 = high school, 4 = others, 5 = unknown, 6 = unknown)\n",
+    "- *marriage*: Marital status (1 = married, 2 = single, 3 = others)\n",
+    "- *age*: Age in years\n",
+    "\n",
+    "**Target / Label**\n",
+    "- *defaulted*: Default payment (1=yes, 0=no)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Loading dataset\n",
+    "\n",
+    "We will use pandas and it's functions to load and maintain data in the notebook. Reading the data into a pandas dataframe using read_csv() function of pandas."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = pd.read_csv(\"defaults.csv\")"
```

Comment:
Please load the dataset from the correct relative path in the repo so that it runs without modification

Reply:
I have done this.